global/certbot-dns-plugins.js

```diff
@@ -284,6 +284,15 @@ dns_godaddy_key = abcdef0123456789abcdef01234567abcdef0123`,
 		credentials:         'dns_hetzner_api_token = 0123456789abcdef0123456789abcdef',
 		full_plugin_name:    'dns-hetzner',
 	},
+        //####################################################//
+        hostingde: {
+                display_name:        'Hosting.de (Experimental)',
+                package_name:        'no-hostingde-api',
+                version_requirement: '~=0.3.1',
+                dependencies:        '',
+                credentials:         'no_hostingde_api:dns_hostingde_apikey = XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
+                full_plugin_name:    'no-hostingde-api:dns-hostingde',
+        },
```

Comment:
Although it is already quite old:
The bot raises the error, because the indentation here is with spaces instead of tabs